doc/source/admin/interactive_environments.rst

```diff
@@ -181,8 +181,8 @@ Please note you will need to be using apache2.4 with ``mod_proxy_wstunnel``.
     }
 
     # Project Jupyter / IPython specific. Other IEs may require their own routes.
-    location /galaxy/gie_proxy/ipython/api/kernels {
-        proxy_pass http://localhost:8800/galaxy/gie_proxy/ipython/api/kernels;
+    location ~ ^/galaxy/gie_proxy/jupyter/(?<nbtype>[^/]+)/api/kernels(?<rest>.*?)$ {
+        proxy_pass http://localhost:8800/galaxy/gie_proxy/$nbtype/api/kernels$rest;
```

Comment:
@bgruening, @pvanheus says this is correct and I believe him, but just wanted a second opinion :)